lib/sidekiq/statistic/views/realtime_statistic.js

```diff
@@ -30,8 +30,16 @@ $(function () {
   var interval
   var excludedWorkers = []
 
+  var toggleVisibilityButton = '.worker__toggle-visibility'
```

Comment:
I know the project isn't using, but shouldn't we start using es6?

Reply:
I was thinking about that when I was doing this feature. Maybe we can do a refactor to use es6. What do you think @wenderjean ?

Reply:
We can do that even though I believe it would be better to do it in a separate ticket, for the purpose of this one it looks good to go.